test/integration/targets/r_cran/tasks/main.yaml

```diff
@@ -0,0 +1,45 @@
+- name: Install 1 package
+  r_cran:
```

Comment:
Integration tests are failing because the module name, `r_pkg_cran`, is different than what is specified in the tasks.

Reply:
should be fixed via bed6c2d080d9ce94abd611b58b8f17fccac1e819